Cargo.toml

```diff
@@ -40,7 +40,16 @@ termcolor = "1.1.2"
 html-escape = { version = "0.2.13", optional = true }
 comrak = { version = "0.20.0", optional = true, default-features = false }
 handlebars = { version = "5.0", optional = true, features = ["string_helpers"] }
-syntect = { version = "5.1.0", optional = true, default-features = false, features = [ "parsing", "default-syntaxes", "default-themes", "html", "dump-load", "regex-onig" ] }
+tree-sitter-highlight = { version = "0.20.1", optional = true }
+
+tree-sitter-javascript = { version = "0.20.1", optional = true }
+tree-sitter-typescript = { version = "0.20.3", optional = true }
+tree-sitter-json = { version = "0.20.1", optional = true }
+tree-sitter-regex = { version = "0.20.0", optional = true }
+tree-sitter-css = { version = "0.20.0", optional = true }
+tree-sitter-toml = { version = "0.20.0", optional = true }
+tree-sitter-md = { version = "0.1.7", optional = true }
```

Comment:
seems inferior in terms of supported languages compared to syntect https://github.com/sublimehq/Packages/tree/fa6b8629c95041bf262d4c1dab95c456a0530122 (though i will admit we dont need all these).
does tree-sitter-json  support jsonc?

Reply:
Yes: 
![image](https://github.com/denoland/deno_doc/assets/7829205/1f51dda2-70f2-42ed-bc96-4731f0c0b114)

Reply:
The main one that is missing right now that I'd like to have is HTML (this is waiting on a crates.io release: https://github.com/tree-sitter/tree-sitter-html/issues/68).

I couldn't think of other languages that may be useful. But you can find the full list here: https://tree-sitter.github.io/tree-sitter/#parsers

Reply:
probably should have wgsl, and rust (and WAT in the future) for wasm modules